k-distribution/k-tutorial/1_basic/19_debugging/README.md

```diff
@@ -169,7 +168,7 @@ line of code containing the function or rule.
 
 For example, consider the following K definition (`lesson-19-b.k`):
 
-```k
+```{.k .numberLines .line-anchors startFrom="1"}
```

Comment:
These don't seem to get rendered on GitHub for me.

Reply:
@fiedlr These changes also break the K Markdown parser, unfortunately

Reply:
@Baltoli That's a shame, it works with Pandoc as a way to add additional classNames and attributes to the code block. Is there a way to do this on our website and not break the K Parser? My point is that it's not ideal to reference line numbers without actually displaying line numbers.

Reply:
Yeah I totally agree with the sentiment; changing the Markdown parser to support this is likely to be a bigger job in the frontend (if it's even possible with the library we use). Is there a way to rewrite the surrounding text instead?

Reply:
I wrote the [parser for the tags](https://github.com/runtimeverification/k/blob/master/kernel/src/main/jjtree/TagSelector.jjt), so we can do whatever we want there.
Can the website interpret that line? We might be able to update the parser in a way that is backward compatible.

Reply:
@Baltoli depending on what we want here, we might just need to update the regex [here](https://github.com/runtimeverification/k/blob/master/kernel/src/main/jjtree/TagSelector.jjt#L144)
It might also be a great first task for @Robertorosmaninho 
The changes are limited to a single file. We have unit tests, and integration tests to update.
I guess the question remains, is the website going to display what we want?

Reply:
@Baltoli @radumereuta If I checked correctly, we use `markdown-it` for generating HTML from `md`. It doesn't seem to work on their live website https://markdown-it.github.io/

I'll investigate if it's possible to do it another way with this.

Reply:
There's a `markdown-it` plugin https://www.npmjs.com/package/markdown-it-plugins that claims to have line-numbers, so it should not be a huge change. Styling can be addressed in the CSS.
@yiyi-wang-rv

Reply:
@yiyi-wang-rv can this actually be done entirely through CSS, do you know?